Microsoft.Identity.Web/StartupHelpers.cs

```diff
@@ -67,7 +67,7 @@ public static IServiceCollection AddAzureAdV2Authentication(this IServiceCollect
                 // If you want to restrict the users that can sign-in to several organizations
                 // Set the tenant value in the appsettings.json file to 'organizations', and add the
                 // issuers you want to accept to options.TokenValidationParameters.ValidIssuers collection
-                options.TokenValidationParameters.IssuerValidator = AadIssuerValidator.GetIssuerValidator(options.Authority).ValidateAadIssuer;
+                options.TokenValidationParameters.IssuerValidator = AadIssuerValidator.GetIssuerValidator(options.Authority).Validate;
```

Comment:
Minor rename, I can revert it. But we will have an interface for all the validators in the future. AadIssuerValidateor.ValidateAadIssuer() sounds oximoronic.